SU-SWS/ood_giving_site
# ADAPT-615 - Story Card

# READY
- (Edit the above to reflect status)

# Summary
- Story Card component - vertical or horizontal
- Row with 2 columns component - column width ratios, row width, spacing below options, item flex alignment options (top, bottom, center, stretch), options to go to 1 column at MD breakpoint (otherwise 1 column at SM breakpoint).
- Update Row with 3 columns component

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull this branch and do `nvm use`, then `gatsby develop`
1. Go to the OOD Giving storyblok space, go to the page "About" -> "Contact Us", scroll down to see the Story Cards.

What it should look like
![Screen Shot 2020-09-25 at 11 23 53 AM](https://user-images.githubusercontent.com/42749717/94302793-a5453000-ff21-11ea-8317-607afece5e7f.png)
![Screen Shot 2020-09-25 at 9 42 36 AM](https://user-images.githubusercontent.com/42749717/94302805-ab3b1100-ff21-11ea-8e5b-9aba889e27b5.png)



# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
https://stanfordits.atlassian.net/browse/ADAPT-643


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
